Jellyfin.Plugin.PlaybackReporting/Pages/user_playback_report.js

```diff
@@ -484,7 +484,6 @@ define(['libraryMenu'], function (libraryMenu) {
 
                     var weeks = view.querySelector('#weeks');
                     weeks.addEventListener("change", process_click);
-                    var days = parseInt(weeks.value) * 7;
 
                     var url = "user_usage_stats/PlayActivity?filter=" + filter_names.join(",") + "&days=" + days + "&end_date=" + end_date.value + "&data_type=count&stamp=" + new Date().getTime();
```

Comment:
but here `days` is used :confused:

Reply:
its used only once now. How it used to be originally, no all option just 1-12 until a way to merge weeks into months so it displays better, right not is not ideal for that page.

Reply:
I mean `"&days=" + days + "&` and `var days` is inside `process_click`.
I think you should leave `var days = parseInt(weeks.value) * 7;` here and update `days` in `process_click` with new `weeks.value`.

Reply:
can you request the change again please.

Reply:
> can you request the change again please.

Which changes? Now it looks fine to me.
Maybe need to make some styling fixes (like space between `){`) and remove extra empty lines.

Reply:
i reverted before that my bad. 

thanks.